ship/src/test/scala/ch/epfl/bluebrain/nexus/ship/RunShipSuite.scala

```diff
@@ -106,20 +105,20 @@ object RunShipSuite {
     override def listObjectsV2(bucket: String): IO[ListObjectsV2Response] =
       IO.raiseError(new NotImplementedError("listObjectsV2 is not implemented"))
 
-    override def listObjectsV2(bucket: Models.BucketName, prefix: String): IO[ListObjectsV2Response] =
+    override def listObjectsV2(bucket: String, prefix: String): IO[ListObjectsV2Response] =
       IO.raiseError(new NotImplementedError("listObjectsV2 is not implemented"))
 
-    override def readFile(bucket: Models.BucketName, fileKey: Models.FileKey): fs2.Stream[IO, Byte] =
+    override def readFile(bucket: String, fileKey: String): fs2.Stream[IO, Byte] =
       fs2.Stream.empty
 
     override def headObject(bucket: String, key: String): IO[HeadObject] =
       IO.raiseError(new NotImplementedError("headObject is not implemented"))
 
     override def copyObject(
-        sourceBucket: Models.BucketName,
-        sourceKey: Models.FileKey,
-        destinationBucket: Models.BucketName,
-        destinationKey: Models.FileKey,
+        sourceBucket: String,
```

Comment:
It would still be nice to reintroduce something more refined than strings at some point

Reply:
I think the problem with these types specifically is when you needed to use them as a string you had to call '.value.value' which is both annoying, and if you forget to do it in string interpolation for example you get a bad result. There might be other options which don't have these issues